src/openrct2/ride/Track.cpp

```diff
@@ -899,3 +899,114 @@ void TrackElement::SetHighlight(bool on)
     if (on)
         Flags2 |= TRACK_ELEMENT_FLAGS2_HIGHLIGHT;
 }
+
+bool TrackSupportsHasFlag(uint8_t queryFlag, uint8_t sequenceFlags)
```

Comment:
if a function is purely only used in the singular .cpp file and has no declaration in a header then it is `static`. So just add `static` before the bool. Same goes to the other function in this file that the compiler is complaining about.